### ISSUE TITLE
question!!

### ISSUE THREAD

**Issue by kdc429** [ABausG/home_widget#226]
This library doesn't have the ability to draw a screen, right?

I currently want to implant the Dart file into the appWidget, but there seems to be no way.

**Comment by MuhammadAbdullah7** [comment]
Yes as mentioned [here](https://github.com/ABausG/home_widget) the purpose of this is for sending data from flutter to widgets and updating them, for how to implement widgets you have to work on native side for both [IOS](https://medium.com/@tiaanvdr55/build-ios-home-screen-widgets-with-flutter-bebe71fa7ec9) and [Android](https://developer.android.com/develop/ui/views/appwidgets).
